js/repl/Repl.js

```diff
@@ -581,6 +591,7 @@ class Repl extends React.Component<Props, State> {
       browsers: envConfig.browsers,
       bugfixes: envConfig.isBugfixesEnabled,
       build: state.babel.build,
+      assumptions: JSON.stringify(envConfig.assumptions),
```

Comment:
We should also add `assumptions` to `URL_KEYS`:

https://github.com/babel/website/blob/70fe2426dac96f524638f18bf1955459d37b8929/js/repl/UriUtils.js#L7

Reply:
yup, I have it there https://github.com/babel/website/pull/2539/files#diff-086d46e0a51a52ba06a867e04c535506c6a37540e5b335d2504b1c4895ed4510R29